calibrated: 1
equilibrium:
    sign: -1
    signFlexor: -1
    signExtensor: -1
    nameFlexor: "wrist_flexor"
    nameExtensor: "wrist_extensor"
    servoRange: 5.4978
    servoOffset: 0
signEncoder: -1
signJoint: 1
name: "wrist"
nameEncoder: "wrist_encoder"
minAngle: -0.75
maxAngle: 0.5
maxAbsForwardError: 5.0
gains:
    P: 0.055
    I: 0.0005
    D: 0.2